scripts/mech_events.py

```diff
@@ -51,15 +53,12 @@
     "Content-Type": "application/json",
 }
 QUERY_BATCH_SIZE = 1000
-MECH_EVENTS_SUBGRAPH_QUERY = Template(
+MECH_EVENTS_SUBGRAPH_QUERY_TEMPLATE = Template(
     """
-    query {
+    query mech_events_subgraph_query($sender: Bytes, $id_gt: Bytes, $first: Int)  {
```

Comment:
do we need types here?

Reply:
Those types are part of the query sent to the Graph. I am unsure if they are strictly necessary, but keeping them does not harm in any case.